package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "impression-tracker-react-hook",
-  "version": "7.4.0",
+  "version": "7.6.0",
```

Comment:
I lost track, do we need to go from 4 all the way to 6?

Reply:
Yea.  It's currently at 7.5.0.  This is a feature update so it bumps the minor version.
https://www.npmjs.com/package/impression-tracker-react-hook